Documentation/ceph-upgrade.md

```diff
@@ -369,8 +409,9 @@ Verify the Ceph cluster's health using the [health verification section](#health
 
 ## 6. CSI Manual Update (optional)
```

Comment:
Again, I think we can remove this section since it was addressed in the 1.2 upgrade

Reply:
Removed.